src/App.vue

```diff
@@ -65,6 +71,24 @@ export default {
       this.updateLogo()
       this.updateOptions()
     })
+
+    // Sound default on
+    let localStorageSound = window.localStorage.getItem('sound')
```

Comment:
Can you move these to a `initializeSounds` etc method, so `created` method would not become more complex